packages/db/package.json

```
@@ -1,8 +1,15 @@
 {
   "name": "@astrojs/db",
   "version": "0.10.3",
-  "description": "",
+  "description": "Add libSQL and Astro Studio support to your Astro site.",
```

Micro-nit: just realised other packages don’t end the description with a `.`
```suggestion
  "description": "Add libSQL and Astro Studio support to your Astro site",
```